src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Aapt2.targets

```diff
@@ -0,0 +1,168 @@
+<!--
+***********************************************************************************************
+Xamarin.Android.Common.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+  created a backup copy.  Incorrect changes to this file will make it
+  impossible to load or build your projects from the command-line or the IDE.
+
+This file imports the version- and platform-specific targets for the project importing
+this file. This file also defines targets to produce an error if the specified targets
+file does not exist, but the project is built anyway (command-line or IDE build).
+
+Copyright (C) 2010-2011 Novell. All rights reserved.
+Copyright (C) 2011-2012 Xamarin. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+<Target Name="_ReadAapt2VersionCache">
+	<ReadLinesFromFile File="$(_AndroidAapt2VersionFile)"
+			Condition="Exists('$(_AndroidAapt2VersionFile)')">
+		<Output TaskParameter="Lines" ItemName="_Aapt2VersionCache"/>
+	</ReadLinesFromFile>
```

Comment:
If we ship `aapt2`, can we just remove all the logic checking the version of `aapt2`?

Reply:
no we still need to check if the version of `aapt2` changed. This is so we can regenerate the `flata` archives. The format of those is internal and subject to change by google. So if the version changes we probably cannot use the old files.